packages/indexer-common/src/allocations/__tests__/tap.test.ts

```diff
@@ -125,25 +151,209 @@ describe('TAP', () => {
     timeout,
   )
 
+  test('should revert the rav request', async () => {
```

Comment:
Would that be more accurate?
```suggestion
  test('`revertRavsRedeemed` should revert RAV redeem status in DB only if older than subgraph last block', async () => {
```

Reply:
Seems more accurate indeed, updated!